src/ObjLoading/Game/T6/BSP/LoaderBSP_T6.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "Asset/IAssetCreator.h"
+#include "Game/T6/T6.h"
```

Comment:
nit: This header seems unused

adjusted in a8fc10df74a0acb264894edbf7988fde88ea0d85